scripts/partition_manager/partition_manager.rst

```diff
@@ -527,6 +530,14 @@ See :ref:`ug_bootloader_external_flash` for more details on using external flash
 
 .. _pm_build_system:
 
+Flash partitions requre device drivers to be enabled for them. The ``DEFAULT_DRIVER_KCONFIG``
+parameter should be set to a Kconfig option that enables the driver during compilation.
```

Comment:
```suggestion
You must enable device drivers for each flash partition. 
To do so, set the ``DEFAULT_DRIVER_KCONFIG`` parameter to the Kconfig option that enables the driver during compilation.
```

Could we be more explicit on indicating here which kconfig option enables the driver during compilation?